inc/motors.h

```diff
@@ -0,0 +1,26 @@
+enum motors{
```

Comment:
First few lines of this file should be:
`#ifndef MOTORS_H_`  
`#define MOTORS_H_`

This will make sure that you don't accidentally define all this stuff twice, which can cause compiler errors. This can happen if File A includes motor.h, and File B includes both File A and motor.h. Then motor.h will be defined twice. To avoid this, you use these #ifndef stuff that will only add the following lines if the define statement hasn't been defined yet. The file should also end with an `#endif`